File: lab4/java/comp26120/bad1.java
package comp26120;

public class bad1 implements PriorityQueue {
    String value;

    public boolean contains(String value, int priority) {
	return false;
    }

    public void insert(String value, int priority) {
    }

    public boolean is_empty() {
	return false;
    }

    public String pop_min() {
	return null;
    }

    public void print() {}
	
}

File: lab4/c/bad1.c
// Giles Reger, 2019

#include <stdio.h>
#include <stdlib.h>
#include <ctype.h>
#include <string.h>
#include <limits.h>
#include <stdbool.h>

#include "pq.h"


struct llist {
  Value_Type value;
};

struct llist* initialize_pq (int size){
  return NULL;
}     

void tidy (struct llist* list){
}

bool contains(struct llist* pq, Value_Type value, int priority){
  return false;
}

void insert(struct llist* list, Value_Type value, int priority){
}

bool is_empty(struct llist* pq){ 
  return false;
}

Value_Type pop_min(struct llist* pq){
  return NULL;
}

void print(struct llist* list){
}

